sql-statements/sql-statement-alter-database.md

```diff
@@ -30,6 +30,10 @@ alter_specification:
 
 The `alter_specification` option specifies the `CHARACTER SET` and `COLLATE` of a specified database. Currently, TiDB only supports some character sets and collations. See [Character Set and Collation Support](/character-set-and-collation.md) for details.
 
+## MySQL compatibility
+
+This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```

Comment:
how about directly use https://github.com/pingcap/tidb/issues/new/choose?

Reply:
I think that makes sense. It looks like [report-issue](https://docs.pingcap.com/tidb/stable/report-issue) has been removed from the TOC, which is also fine by me.

Reply:
This has been fixed.